cli_client/python/timesketch_cli_client/commands/timelines.py

```diff
@@ -52,3 +52,43 @@ def describe_timeline(ctx, timeline_id):
         return
     click.echo(f"Name: {timeline.name}")
     click.echo(f"Index: {timeline.index_name}")
+
+
+@timelines_group.command("event-add-tags")
+@click.argument("timeline_id", type=int, required=False)
+@click.option("--event_id", required=True, help="ID of the event.")
+@click.option(
+    "--tags",
+    required=True,
+    help="Comma seperated list of Tags to add to the event.",
+)
+@click.option(
```

Comment:
drive by review, I know this has already been merged.

Whey is --output-format needed here? It has nothing to do with adding tags.

Please followup and remove if there isn't anything I'm missing here.

Reply:
This is to be able to tweak the output-format to JSON if the user wants to use the output of the sommand somewhere else.